clientd3d/preferences.h

```diff
@@ -0,0 +1,8 @@
+#ifndef _PREFERENCES_H
+#define _PREFERENCES_H
+
+#pragma once
+#include <windows.h>
```

Comment:
Remove this include; the client relies on every source file including "client.h", which then includes windows.h.